client/js/views/Stats.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div id="stats-wrapper">
+    <!-- Upper Service -->
+    <section id="upper-surface" class="pa-8 mb-6">
+      <v-select
+        v-model="selectedSeason"
+        :items="seasons"
+        label="Select Season"
+        data-cy="season-select"
+        return-object
+        item-text="name"
+      >
+        <template #selection="{ item }">
+          <h1 class="text-h2" data-cy="selected-season-header">
+            {{ item.name }}
+          </h1>
+        </template>
+      </v-select>
+      <h4 v-if="selectedSeason" class="text-h4">
+        <span data-cy="season-start-date">{{ seasonStartString }}</span>
+        -
+        <span data-cy="season-end-date"> {{ seasonEndString }} </span>
+      </h4>
+    </section>
+    <section class="px-8">
+      <!-- Season Champions -->
+      <div class="mb-10">
+        <h2
+          v-if="
```

Comment:
This seems like it should be a computed prop, something like `showSeasonChampions`.

Reply:
Agreed